code/modules/medical/surgery_procs.dm

```diff
@@ -657,17 +645,44 @@ var/global/list/chestitem_whitelist = list(/obj/item/gnomechompski, /obj/item/gn
 				surgeon, "<span class='alert'>You cut out an implant from [surgeon == patient ? "yourself" : "[patient]"] with [src]!</span>",\
 				patient, "<span class='alert'>[patient == surgeon ? "You cut" : "<b>[surgeon]</b> cuts"] out an implant from you with [src]!</span>")
 
-				var/obj/item/implantcase/newcase = new /obj/item/implantcase(patient.loc, usedimplant = I)
-				I.on_remove(patient)
-				patient.implant.Remove(I)
-				var/image/wadblood = image('icons/obj/surgery.dmi', icon_state = "implantpaper-blood")
-				wadblood.color = patient.blood_color
-				newcase.UpdateOverlays(wadblood, "blood")
-				newcase.blood_DNA = patient.bioHolder.Uid
-				newcase.blood_type = patient.bioHolder.bloodType
+				if (I.shrapnel)
+					patient.implant.Remove(I)
+					I.set_loc(patient)
+					I.on_remove(patient)
+				else
+					var/obj/item/implantcase/newcase = new /obj/item/implantcase(patient.loc, usedimplant = I)
+					I.on_remove(patient)
+					patient.implant.Remove(I)
+					var/image/wadblood = image('icons/obj/surgery.dmi', icon_state = "implantpaper-blood")
+					wadblood.color = patient.blood_color
+					newcase.UpdateOverlays(wadblood, "blood")
+					newcase.blood_DNA = patient.bioHolder.Uid
+					newcase.blood_type = patient.bioHolder.bloodType
+
+				return 1
+
+/* ---------- SCALPEL - CHEST ---------- */
+	// else if (surgeon.zone_sel.selecting == "chest" && (surgeon.a_intent == "help" || surgeon.a_intent == "disarm"))
```

Comment:
?

Reply:
This comment and a lot of the code you requested a change for isn't something I wrote, it's me reordering and moving around the old surgery code. I'll refactor it so it's clean, but I don't actually have an answer for your confusion.